docs/release/release_0_3_6.md

```diff
@@ -0,0 +1,108 @@
+# napari 0.3.6
+
+We're happy to announce the release of napari 0.3.6!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+This release contains the long awaited addition of text to both the points and
+shapes layers (#1374). Checkout our `examples/*_with_text.py` for simple usage
+and this great [segmentation annotation tutorial](https://napari.org/tutorials/applications/annotate_segmentation)
+for a more real-world use case. We've also added support for a circular
+paintbrush for easier labels painting, and moved more of our contrast limits
+and gamma setting to the GPU for faster rendering and interactivity with
+3D rendered datasets. As always this release contains various bug fixes and
+other improvements.
```

Comment:
```suggestion
This release contains the long awaited addition of text to both the points and
shapes layers (#1374). Checkout our `examples/*_with_text.py` for simple usage
and this great [segmentation annotation tutorial](https://napari.org/tutorials/applications/annotate_segmentation) for a more real-world use case.

We've also added support for a circular
paintbrush for easier labels painting, and moved more of our contrast limits
and gamma setting to the GPU for faster rendering and interactivity with
3D rendered datasets. As always this release contains various bug fixes and
other improvements.
```

Reply:
I think putting the circular paintbrush in a separate paragraph helps highlight it. I don't have a strong preference either way though.

Reply:
Good idea!